src/marqo/upgrades/v20_v21.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from marqo import logging, version
+from marqo.api import exceptions as api_exceptions
+from marqo.core.index_management.index_management import IndexManagement, _MarqoConfig
+from marqo.upgrades.upgrade import Upgrade
+from marqo.vespa.models import VespaDocument
+from marqo.vespa.vespa_client import VespaClient
+
+logger = logging.get_logger(__name__)
+
+
+class v20v21(Upgrade):
+    """
+    Upgrade from Marqo v2.0 to v2.1.
+
+    The migration comprises two steps:
+    1. Create the default query profile
+    2. Add Marqo version to Marqo settings schema
+    """
```

Comment:
Do we also need validation step that, after the pushes have been completed, ensures that the query profile exists and the version is as expected? (I realise this extra check is heading in the direction of over-engineering, and purely reducing the chances of inconsistencies rather than guaranteeing them, but it could still be useful for such a signifcant change). We don't necessarily need to include this in this release

Reply:
Added